README.md

```diff
@@ -33,21 +33,19 @@ Default conda channel:
 * [`jupyterlab`](https://jupyterlab.readthedocs.io/en/stable/)
 * [`nglview`](http://nglviewer.org/nglview/latest/)
 
-#### Known Bugs
-
-`nglview` does not work in non-Linux environments (Windows & MacOS).
-
 ## Getting Started
 
 ### Linux
 
 1. Install [Anaconda](https://www.anaconda.com/download/) for Python 3.6 or higher
 2. Execute the `setup-conda-linux.sh` script which will automatically set up a Python environment named `WTcourse`
 3. Activate the conda environment with: `source activate WTcourse`
+4. Lunch [Jupyter Notebook](https://jupyter.org/) with `jupyter notebook --NotebookApp.iopub_data_rate_limit=10000000`
```

Comment:
Can this be changed to Launch instead of Lunch? (same thing for the section below).

Reply:
Wooops, fixed.